backend/Dockerfile

```diff
@@ -2,15 +2,31 @@ FROM mavromat/alpine-python:3.6
 
 LABEL maintainer="Dimitrios Mavrommatis <jim.mavrommatis@gmail.com>"
 
-RUN apk update && apk add --no-cache supervisor tcl postgresql-client postgresql-dev redis curl
+RUN apk update && apk add --no-cache supervisor tcl postgresql-client postgresql-dev redis curl git libffi-dev cmake libssh
 
 WORKDIR /root
 
 COPY . ./
 
+RUN git clone https://github.com/rtrlib/rtrlib.git
```

Comment:
this always downloads the latest, maybe version lock it?

Reply:
how?

Reply:
done, resolving.